test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,62 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+require('dotenv').config();
+
+chai.use(chaiSubset);
+
+const instance = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+const urlBase = 'https://api.github.com/gists';
+const promise = 'new Promise((resolve, reject) => {resolve(123)})';
+let createdGist;
+let getGist;
+let deleteGist;
+
+const gist = {
+  description: 'Creating a Gist',
+  public: true,
+  files: {
+    'Promise.js': {
+      content: promise
+    }
+  }
+};
+
+describe('Create and Delete a Gist', () => {
+  it('Create a Gist', async () => {
+    createdGist = await instance.post(`${urlBase}`, gist);
+
+    expect(createdGist.status).to.equal(StatusCodes.CREATED);
+    expect(createdGist.data.description).to.equal(gist.description);
+    expect(createdGist.data.public).to.equal(gist.public);
+    expect(createdGist.data).to.containSubset(gist);
+  });
+
+  it('Get a Gist', async () => {
+    getGist = await instance.get(createdGist.data.url);
+
+    expect(getGist.status).to.equal(StatusCodes.OK);
+  });
+
+  it('Delete a Gist', async () => {
+    deleteGist = await instance.delete(createdGist.data.url);
+
+    expect(deleteGist.status).to.equal(StatusCodes.NO_CONTENT);
+  });
+});
+
+describe('Check deletion Gist', () => {
```

Comment:
This describe should be inside the others, for the same reason explained on the past its: you are using a variable createdGist, so if you execute this describe only, it will fail